index.html

```diff
@@ -8,24 +8,27 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="style2.css">
     <link rel="stylesheet" href="style3.css">
+    
 </head>
 <body>
     <header>
+        <div class="mob-background"></div>
         <nav class="navbar">
             <div class="wlc">Welcome</div>
             <div class="menu-icon"> <img src="images/Icon-Menu.svg" alt="menu-icon"></div>
             
             <ul class="nav-list nav-desktop">
-                <li><a href="">Porfolio</a></li>
-                <li><a href="">About</a></li>
-                <li><a href="">Contact</a></li>
+                <li class="linkclass"><a href="#portfolio">Porfolio</a></li>
+                <li class="linkclass"><a href="#about">About</a></li>
+                <li class="linkclass"><a href="#contact">Contact</a></li>
+                <li class="aa closeclass">&times;</li>
             </ul>
         </nav>
```

Comment:
- [ ] Nice job so far, your mobile menu works:+1:  but there is one more thing you need to do :cry: There is a small issue with the design of the bottom of your menu with `border-radius-bottom-left` and `border-radius-bottom-right`, to view as in the drawing of Figma, I have to scroll from right to left. Please fix it to make it similar to the Figma template. :smile: 
### Figma template
![Screenshot from 2022-07-19 20-45-15](https://user-images.githubusercontent.com/6615394/179849920-0b0f0749-b338-415f-83d7-4befbc01091d.png)

### Your work
![Screenshot from 2022-07-19 20-44-26](https://user-images.githubusercontent.com/6615394/179849927-9b21d287-6dab-4a23-bb7b-4bd387a36080.png)

- [ ]  Please adjust the `opacity` of the empty space after the end of the mobile menu so we can't see the social network icons in the background like on the Figma template.

https://www.loom.com/share/c4ef64fae2d64d299ab9185886f9c45a

Reply:
All changes have been implemented